components/02-elements/list/list.config.js

```diff
@@ -66,6 +66,18 @@ module.exports = {
       }
     },
     {
+      name: 'center',
+      context: {
+        modificators: 'list--center'
+      }
+    },
+    {
+      name: 'center-horizontal',
+      context: {
+        modificators: 'list--center list--horizontal'
```

Comment:
You are defining `modificators` in config but using `class` in template. Rename everyting to `class`.